src/fsharp/FSComp.txt

```diff
@@ -1632,3 +1632,4 @@ reprStateMachineInvalidForm,"The state machine has an unexpected form"
 3518,implMissingInlineIfLambda,"The 'InlineIfLambda' attribute is present in the signature but not the implementation."
 3519,tcInlineIfLambdaUsedOnNonInlineFunctionOrMethod,"The 'InlineIfLambda' attribute may only be used on parameters of inlined functions of methods whose type is a function or F# delegate type."
 3520,invalidXmlDocPosition,"XML comment is not placed on a valid language element."
+3521,tcOnlySimplePatternsInMemberConstructs,"Only simple variable patterns can be bound in 'member' constructs"
```

Comment:
It feels like the diagnostic is unclear - beginners won't understand this.  Perhaps  `Invalid member declaration. The name of the member is missing.`

Reply:
I think still unclear
`The name of the member is missing` is not actually true

What about `Invalid member declaration. The name of the member shouldn't be parenthesized`?

Reply:
The problem is that we don't know that the thing in the parentheses is the name.  I guess `Invalid member declaration. The name of the member is missing or has parentheses.`

Reply:
Got it